demos/DemoSearchComponentFour.module.scss

```diff
@@ -147,10 +147,14 @@
   display: grid;
   grid-template-columns: repeat(4, 1fr);
   font-size: 12px;
+    @media (max-width: 768px) {
+    grid-template-columns: 1fr;
+  }
```

Comment:
Looks good, but might need to pop this out into it's own CSS rule, because there are still a few old browsers that don't support nested media queries: https://caniuse.com/mdn-css_at-rules_media_nested-queries

Though the only one I'm concerned about is IE 11, but it's such a small percentage that I think we can ignore it. Should be safe for now.

Reply:
Though make sure to properly align the query itself and provide a bit of padding:
```suggestion
  
  @media (max-width: 768px) {
    grid-template-columns: 1fr;
  }
```